### PR TITLE
Created Chaska file.

### PR DESCRIPTION
**Pull request by tlcoder**
Look ok?

### CODE REVIEW

File: tony-lein/chaska.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Chaska
+
+### Restaurants :pizza:
+- Pizza Hut
```
Comment (shadow247) [inline tony-lein/chaska.md:4]: @tony-lein Any local (non-chain) restaurants???

File: tony-lein/chaska.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+- Domino's
+- Papa John's
+
```
Comment (ksolissprc) [inline tony-lein/chaska.md:7]: @tony-lein Which is your favorite?

File: tony-lein/chaska.md (hunk 1 above)
Comment (tlcoder) [inline tony-lein/chaska.md:4]: No

File: tony-lein/chaska.md (hunk 2 above)
Comment (tlcoder) [inline tony-lein/chaska.md:7]: Don't like pizza.

**Pull request merged**